### PR TITLE
Plane: change CURRENT to BATT_MON in log bitmask metadata

### PR DESCRIPTION
**Pull request by Hwurzburg**
NFC last cleanup of Plane logging

### CODE REVIEW

File: ArduPlane/Parameters.cpp [hunk 1]
```diff
@@ -618,7 +618,7 @@ const AP_Param::Info Plane::var_info[] = {
     // @Param: LOG_BITMASK
     // @DisplayName: Log bitmask
     // @Description: Bitmap of what on-board log types to enable. This value is made up of the sum of each of the log types you want to be saved. It is usually best just to enable all log types by setting this to 65535. The individual bits are ATTITUDE_FAST=1, ATTITUDE_MEDIUM=2, GPS=4, PerformanceMonitoring=8, ControlTuning=16, NavigationTuning=32, Mode=64, IMU=128, Commands=256, Battery=512, Compass=1024, TECS=2048, Camera=4096, RCandServo=8192, Sonar=16384, Arming=32768, FullLogs=65535
-    // @Bitmask: 0:ATTITUDE_FAST,1:ATTITUDE_MED,2:GPS,3:PM,4:CTUN,5:NTUN,6:MODE,7:IMU,8:CMD,9:CURRENT,10:COMPASS,11:TECS,12:CAMERA,13:RC,14:SONAR,15:ARM/DISARM,19:IMU_RAW,20:ATTITUDE_FULLRATE,21:VideoStabilization
+    // @Bitmask: 0:ATTITUDE_FAST,1:ATTITUDE_MED,2:GPS,3:PM,4:CTUN,5:NTUN,6:MODE,7:IMU,8:CMD,9:BATT_MON,10:COMPASS,11:TECS,12:CAMERA,13:RC,14:RANGEFINDER,15:ARM/DISARM,19:IMU_RAW,20:ATTITUDE_FULLRATE,21:VideoStabilization
```
Comment (IamPete1) [inline ArduPlane/Parameters.cpp:621]: Might as well remove MODE, TECS and ARM/DISARM. Any reason to be all capitals, I think it makes it harder to read.

**Review by WickedShell — COMMENTED** [review]
Not a blocker, as the change is similar to the rest (and valid), but maybe we should just do a clean up pass and humanize these a bit more? For example instead of `BATT_MON` which might be confusing to users, why not just call it `Battery` or something along those lines? It is scope creep though.

File: ArduPlane/Parameters.cpp (hunk 1 above)
Comment (Hwurzburg) [reply ArduPlane/Parameters.cpp:621]: all the others are all caps...existing pattern and matches search for mask bits

Comment (WickedShell) [reply ArduPlane/Parameters.cpp:621]: I'd also support the comment that just breaking the caps lock is more humane.

**Review by tridge — APPROVED** [review]

File: ArduPlane/Parameters.cpp (hunk 1 above)
Comment (TunaLobster) [inline ArduPlane/Parameters.cpp:621]:
```suggestion
    // @Bitmask: 0:ATTITUDE_FULL_RATE,1:ATTITUDE_MED,2:GPS,3:PM,4:CTUN,5:NTUN,6:MODE,7:IMU,8:CMD,9:BATT_MON,10:COMPASS,11:TECS,12:CAMERA,13:RC,14:RANGEFINDER,15:ARM/DISARM,19:IMU_RAW,20:ATTITUDE_FULLRATE,21:VideoStabilization
```

**Pull request merged**